eoscostarica/guias.eoscostarica.io
# docs(frontend):MetodologiasAgiles-195

### GH Issue

Resolves #195  

### Steps to test
1. Go to Metodologías Agiles
1. Check content is right.

#### CheckList
- [ ] The content is similar to its English version.
